composer.json

```diff
@@ -12,7 +12,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "ext-mbstring": "*"
```

Comment:
We could then check if the mb-string is enabled in these rules and throw a `RuntimeException` telling it's required. I think it's better than a Fatal Error...

What do you guys think?